pymtl3/passes/VcdGenerationPass.py

```diff
@@ -201,12 +143,9 @@ def to_bits( obj ):
       if isinstance( obj, Bits ):
         return obj
       # BitStruct
-      bits = [to_bits(getattr(obj, v)) for v in obj.__bitstruct_fields__.keys()]
-      for _bits in bits[1:]:
-        bits[0] = (bits[0] << _bits.nbits) | _bits
-      return bits[0]
+      return concat( *[ to_bits(getattr(obj, v)) for v in obj.__bitstruct_fields__.keys() ] )
```

Comment:
Should we include the to_bits function in pymtl or keep it a user space thing as it is now? I think Kai’s visualization pass needs something similar?

Reply:
yes it does. we can add it to pymtl3.datatypes?